skiko/build-with-local-skia.sh

```diff
@@ -43,7 +43,7 @@ case $SKIA_TARGET in
     ;;
```

Comment:
Let's copy this
https://github.com/JetBrains/skia-pack/blob/8772c12c1d2881f9e663b0dfc8bc264a08a744aa/README.md#windows-specific

To DEVELOPMENT.md requirements

Reply:
Done!